app/src/main/java/app/revanced/manager/ui/screen/settings/AboutSettingsScreen.kt

```diff
@@ -35,43 +33,76 @@ import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.unit.dp
 import app.revanced.manager.BuildConfig
 import app.revanced.manager.R
+import app.revanced.manager.network.dto.ReVancedSocial
 import app.revanced.manager.ui.component.AppTopBar
 import app.revanced.manager.ui.component.ColumnWithScrollbar
 import app.revanced.manager.ui.component.settings.SettingsListItem
+import app.revanced.manager.ui.viewmodel.AboutViewModel
+import app.revanced.manager.ui.viewmodel.AboutViewModel.Companion.getSocialIcon
 import app.revanced.manager.util.isDebuggable
 import app.revanced.manager.util.openUrl
 import com.google.accompanist.drawablepainter.rememberDrawablePainter
+import org.koin.androidx.compose.getViewModel
 
 @OptIn(ExperimentalMaterial3Api::class, ExperimentalLayoutApi::class)
 @Composable
 fun AboutSettingsScreen(
     onBackClick: () -> Unit,
     onContributorsClick: () -> Unit,
     onLicensesClick: () -> Unit,
+    viewModel: AboutViewModel = getViewModel()
 ) {
     val context = LocalContext.current
     // painterResource() is broken on release builds for some reason.
     val icon = rememberDrawablePainter(drawable = remember {
         AppCompatResources.getDrawable(context, R.drawable.ic_logo_ring)
     })
 
-    val filledButton = listOf(
-        Triple(Icons.Outlined.FavoriteBorder, stringResource(R.string.donate)) {
-            context.openUrl("https://revanced.app/donate")
-        },
-        Triple(Icons.Outlined.Language, stringResource(R.string.website), third = {
-            context.openUrl("https://revanced.app")
-        }),
-    )
+    val (preferredSocials, socials) = remember(viewModel.socials) {
+        viewModel.socials.partition(ReVancedSocial::preferred)
+    }
 
-    val outlinedButton = listOf(
-        Triple(Icons.Outlined.Code, stringResource(R.string.github), third = {
-            context.openUrl("https://revanced.app/github")
-        }),
-        Triple(Icons.Outlined.MailOutline, stringResource(R.string.contact), third = {
-            context.openUrl("mailto:nosupport@revanced.app")
-        }),
-    )
+    val donateText = stringResource(R.string.donate)
+    val contactText = stringResource(R.string.contact)
+    val preferredSocialButtons = remember(preferredSocials, viewModel.donate, viewModel.contact) {
+        preferredSocials.map {
+            Triple(
+                getSocialIcon(it.name),
+                it.name,
+                third = {
+                    context.openUrl(it.url)
+                }
+            )
+        } + listOfNotNull(
+            viewModel.donate?.let {
+                Triple(
+                    Icons.Outlined.FavoriteBorder,
+                    donateText,
+                    third = {
+                        context.openUrl(it)
+                    }
+                )
+            }
+        ) + listOfNotNull(
```

Comment:
We only need one list
```suggestion
            },
```